### ISSUE TITLE
Rethink the install of MODX for non technical people

### ISSUE THREAD

**Issue by DESIGNfromWITHIN** [modx-ccc-2015/whishlist#45]

**Comment by mindeffects** [comment]
My idea was a small PHP script that let's you choose what version/type you want, dependig on what your system is capeable of (trad/adv). Download, unpack, setup. Done.
Updates should work from within the system. "Update MODX" inside the manager, close MODX up, get the update, analyses core.config.inc.php to find paths to core, manager etc., run setup without any dialogues (with dialogs, if wanted), done. Simple, right?! :-D

**Comment by JensWolff** [comment]
@mindeffects  +1

**Comment by rthrash** [comment]
Yes please!

**Comment by Gallenkamp** [comment]
Agreed. Maybe even with the option to choose a default template(-package) to start with.

**Comment by adamwintle** [comment]
"Update MODX" inside the manager would be a welcome feature! :+1:

**Comment by JensWolff** [comment]
@adamwintle We were also thinking about that. :+1:
